x-pack/plugins/enterprise_search/server/routes/app_search/index.test.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mockDependencies, MockRouter } from '../../__mocks__';
+
+import { registerAppSearchRoutes } from './';
+
+describe('registerAppSearchRoutes', () => {
+  it('runs without errors', () => {
```

Comment:
These aren't great tests - it basically just confirms that the parent fn runs w/o assertions I'm not sure it's worth the LOE to expand them further since we weren't testing them originally anyway in plugin.ts - and in theory each individual `registerXRoute` should have its own fleshed out unit testing in any case.

Reply:
I ran into this same issue with https://github.com/elastic/kibana/pull/82669 and just replaced the first test in that file with the `registerWSGroupRoutes` and coverage passed but good to know we can do this for coverage purposes.